index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8">
+  <title>JavaScript課題②</title>
+  <style>
+    /* .Num {
+      float: left;
+      clear: both;
+    } */
+    li {
+      list-style: none;
+    }
+  </style>
+</head>
+
+<body>
+  <h1>FizzBuzz問題</h1>
+  <div class="subtitle">FizzNum:
+    <input id="fizzNum" class="Fizz Num" type="text" placeholder="整数値を入力してください" style="width: 170px;">
+
+  </div>
+  <div class="subtitle">BuzzNum:
+    <input id="buzzNum" class="Buzz Num" type="text" placeholder="整数値を入力してください" style="width: 170px;">
+
+  </div>
+  <div>
+    <input id="button" type="submit" value="実行">
+  </div>
+  <p>【出力】</p>
+  <ul id="box"></ul>
```

Comment:
この `box` も抽象度が高く、一見しただけではどんな役割の要素なのかわかりません…

```suggestion
  <ul id="resultBox"></ul>
```